tests/remote-cache/test.sh

```diff
@@ -28,7 +28,7 @@ export REGISTRY="$REGISTRY_IP:5000"
 
 if test -z "$REGISTRY_IP"
 then
-     echo echo "Error: REGISTRY_IP is empty"
+     echo "Error: REGISTRY_IP is empty"
```

Comment:
I've decided to guess that this is errant. Although, it's certainly possible it's constructing a `Dockerfile` where it wants a level of `echo ...`. If that's the case, I'd suggest wrapping the content w/ `'...'`